.github/pull_request_template.md

```diff
@@ -19,8 +19,8 @@
 
 **Notes to PR author**
 
-⚠️  Please make sure the changes are adhere to the guidelines mention [here](https://aligent.atlassian.net/wiki/spaces/AL/pages/2758836297/Development+checklist+FE+BE+INT)
+⚠️ Please make sure the changes are adhered to the guidelines mention [here](https://aligent.atlassian.net/wiki/spaces/AL/pages/2728919167/Pull+Request+guidelines)
```

Comment:
`Please make sure the changes adhere to the guidelines mentioned [here]` - I think this is a bit clearer imo